iroh-bytes/src/store/flat.rs

```diff
@@ -1197,7 +1197,7 @@ impl Store {
         meta_path: PathBuf,
         rt: crate::util::runtime::Handle,
     ) -> anyhow::Result<Self> {
-        tracing::debug!(
+        tracing::info!(
```

Comment:
should this really be `info`?

Reply:
Yes. I constantly need to be reminded where the damn store is so I can nuke it. Compared to all the other stuff we log at info level, this is more important.